test/com/google/enterprise/adaptor/filenet/FileNetAdaptorTest.java

```diff
@@ -254,7 +263,9 @@ public void testInit_objectStore() throws Exception {
     String objectStore = "ObjectStore";
     config.overrideKey("filenet.objectStore", objectStore);
     adaptor.init(context);
-    assertEquals(objectStore, adaptor.getParams().getObjectStore());
+    Connection conn = getConfigOptions().getConnection();
+    // TODO(bmj): use hamcrest.isA
+    assertTrue(getConfigOptions().getObjectStore(conn) instanceof ObjectStore);
```

Comment:
This could hardly fail, since getObjectStore returns an ObjectStore. What is this supposed to be testing.